Okta/OktaTokenManager.swift

```diff
@@ -36,22 +34,21 @@ open class OktaTokenManager: NSObject, NSCoding {
     }
 
     open var idToken: String? {
-        // Return the known idToken via the internal _idToken var
-        // since it gets lost on refresh
+        // Return the known idToken if it is valid
         get {
-            guard let token = self._idToken else { return nil }
-            var returnToken: String?
+            guard let tokenResponse = self.authState.lastTokenResponse,
```

Comment:
Previously, Okta was not returning an `idToken` when a `refreshToken` was exchanged for a new `accessToken`. This required us to store the previous `idToken` separately until [OKTA-136495](https://oktainc.atlassian.net/browse/OKTA-136495) was resolved.

Now, all new tokens are included in the `lastTokenResponse` object.

Reply:
Do we need null checks here for lastTokenResponse?

Reply:
We're OK in this case, as `authState` is a required value in order to have access to the `tokenManager` object.

Also, since `lastTokenResponse` is an optional method - wrapping this in a `guard` statement will fall through to our `else` handler if it returns undefined.